storage/rocksdb/ha_rocksdb.cc

```diff
@@ -44,7 +44,6 @@
 #include "rocksdb/slice_transform.h"
 #include "rocksdb/utilities/checkpoint.h"
 #include "rocksdb/utilities/convenience.h"
-#include "rocksdb/utilities/flashcache.h"
```

Comment:
Hmm, yeah, that is odd. That all should have been merged already and shouldn't appear here as part of the rebased starting point.